index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header>
+            <div class="appbar">
+                <div class="times"><img src="img\Time.png"></div>
+                <div class="group3"><img src="img\Group 3.png"></div>
+            </div>
+            <ul class="nav1">
+                <a href=""><li class="portfolio">Portfolio</li></a>
+                <a href=""><li class="about">About</li></a>
+                <a href=""><li class="contact">Contact</li></a>
+            </ul>
+            <div class="nav2">
+                <div class="mylogo">My Logo</div>
+                <a href=""><i class="bar fa fa-bars"></i></a>
+            </div>
```

Comment:
at this part you should cover with nav and should My logo is first and navitem in nav1 should secold.Example 
<nav>
<div class="nav2">
                <div class="mylogo">My Logo</div>
                <a href=""><i class="bar fa fa-bars"></i></a>
            </div>
<div class ="nav1">
<ul >
                <a href=""><li class="portfolio">Portfolio</li></a>
                <a href=""><li class="about">About</li></a>
                <a href=""><li class="contact">Contact</li></a>
            </ul>
</div>
</nav>